pandas/tests/scalar/timestamp/test_unary_ops.py

```diff
@@ -148,11 +148,13 @@ def test_round_minute_freq(self, test_input, freq, expected, rounder):
         result = func(freq)
         assert result == expected
 
-    def test_ceil(self):
-        dt = Timestamp("20130101 09:10:11")
+    @pytest.mark.parametrize("unit", ["ns", "us", "ms", "s"])
```

Comment:
Could you parameterize `test_ceil` below too?

Reply:
good idea, updated